### PR TITLE
Synchronize allmodules and allmodules+allpatterns

### PR DESCRIPTION
**Pull request by foursixnine**
See bsc#1177321 and bsc#1177446

As far as for the test scenario goes, we only need to ensure that there are no conflicts from the package management level

https://progress.opensuse.org/issues/73627

VR: 
    * [allmodules+allpatterns-gnome@s390x-kvm-sle12](https://openqa.suse.de/t4871240)
    * [allmodules+allpatterns-gnome@ppc64le-2g](https://openqa.suse.de/t4871241)
    * [allpatterns@64bit](https://openqa.suse.de/tests/4871082)

### CODE REVIEW

File: schedule/functional/allmodules+allpatterns-gnome.yaml [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
 description: >
-    Maintainer: zluo
-    Installation with all patterns and all modules without post-installation
-    test modules and it doesn't publish qcow2 image. Exclude modules not
-    designed to work with all other modules together
+    Maintainer: foursixnine.
+    Installation with all patterns selected for installation to check for
+    potential package conflicts, how the system handles big space usage, etc.
+    allpatterns installations can take longer, especially on non-x86_64
+    architectures.
```
Comment (SergioAtSUSE) [inline schedule/functional/allmodules+allpatterns-gnome.yaml:8]: There is information lost. Please, don't remove

- "modules without post-installation test"
- "it doesn't publish qcow2 image."
- "Exclude modules not designed to work with all other modules together"

Comment (foursixnine) [reply schedule/functional/allmodules+allpatterns-gnome.yaml:8]: I'll add back all, but the last line, as it doesn't make sense.

**Comment by foursixnine** [comment]
Just a note here: allpatterns.yaml was never executed on osd (it was using main.pm logic) I've updated the test suite to use it from now on

File: schedule/functional/allpatterns.yaml [hunk 2]
```diff
@@ -1,13 +1,24 @@
 name:           allpatterns
 description:    >
-    Maintainer: slindomansilla.
+    Maintainer: foursixnine.
     Installation with all patterns selected for installation to check for
     potential package conflicts, how the system handles big space usage, etc.
     allpatterns installations can take longer, especially on non-x86_64
-    architectures.
+    architectures. This test suite doesn't publish an HDD, and doesn't run
+    post-installation tests.
```
Comment (SergioAtSUSE) [inline schedule/functional/allpatterns.yaml:8]: ok, after deleting the other YAML, I think we have to add here info about the difference between all patterns from default SLE modules and from selected compatible SLE modules.

Could you add a mention like
```
All SLE modules that are compatible between them can be selected by adding the setting ENABLE_ALL_SCC_MODULES=1
```

Comment (foursixnine) [reply schedule/functional/allpatterns.yaml:8]: @SergioAtSUSE Done

**Review by SergioAtSUSE — APPROVED** [review]

**Comment by SergioAtSUSE** [comment]
Travis needs some love

**Pull request merged**